src/main/java/GenericsAndCllections/User.java

```diff
@@ -0,0 +1,31 @@
+package GenericsAndCllections;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class User {
+    public long userId;
+    public String username;
+    public Map<String, Address> addresses;
+
+    public User(String username, Map<String, Address> addresses) {
+        Random rand = new Random();
+        this.userId = rand.nextInt(999);
+        this.username = username;
+        this.addresses = new HashMap<String, Address>();
+    }
+
+    public User(String username, Address address) {
+        Random rand = new Random();
+        this.userId = rand.nextInt(9999);
+        this.username = username;
+        this.addresses = new HashMap<String, Address>();
+        this.addresses.put("default", address);
+    }
+
+    @Override
+    public String toString() {
+        return "username='" + username;
+    }
```

Comment:
please add a method to add more addresses for your user, for example: addAddress(String key, Address address)

Reply:
done